package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "cho-cho",
+  "version": "1.0.0",
+  "description": "## Instructions",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Ainsley929/last-train-to-nodeville.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Ainsley929/last-train-to-nodeville/issues"
+  },
+  "homepage": "https://github.com/Ainsley929/last-train-to-nodeville#readme",
+  "devDependencies": {
+    "chai": "^4.2.0",
+    "index": "^0.4.0",
+    "mocha": "^6.2.0"
+  },
+  "dependencies": {
```

Comment:
I'm betting you installed this because VS Code complained. So ESLint is a dev tool for maintaining proper code style so it is actually a devDependency too. We'll see that in future exercises when we get a package.json with the dependencies listed. It's not part of this exercise so no worries on not getting it in the right place.